lhotse/recipes/xbmu_amdo31.py

```diff
@@ -0,0 +1,126 @@
+"""
+About the XBMU-AMDO31 corpus
+XBMU-AMDO31 is an open-source Amdo Tibetan speech corpus published by Northwest Minzu University.
+publicly available on https://huggingface.co/datasets/syzym/xbmu_amdo31
+"""
+
+import logging
+import os
+import shutil
+import tarfile
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, Optional, Union
+
+from tqdm.auto import tqdm
+
+from lhotse import validate_recordings_and_supervisions
+from lhotse.audio import Recording, RecordingSet
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike, safe_extract, urlretrieve_progress
+
+
+def download_xbmu_amdo31(
+    target_dir: Pathlike = ".",
+) -> Path:
+    """
+    Downdload and untar the dataset
+    :param target_dir: Pathlike, the path of the dir to storage the dataset.
+    :param force_download: Bool, if True, download the tars no matter if the tars exist.
```

Comment:
The `force_download` and `base_url` arguments are not present in the function definition.

Reply:
This part of the code will be updated today or tomorrow, and the downloadable corpus code in icefall will be removed and added to lhotse.